buildSrc/build.gradle

```diff
@@ -54,3 +72,20 @@ spotless {
     greclipse().configFile('greclipse.properties')
   }
 }
+
+gradlePlugin {
+  plugins {
+    beamModule {
+      id = 'org.apache.beam-module'
```

Comment:
Ah, I didn't know it was so easy! When I was reading about it, I thought you had to actually publish the plugin somewhere in order to use it by id.

Nit: maybe `org.apache.beam.module` and `org.apache.beam.vendor-java` and `org.apache.beam.jenkins`.